cmd/gctcli/commands.go

```diff
@@ -2236,6 +2236,39 @@ var cancelAllOrdersCommand = &cli.Command{
 	},
 }
 
+var modifyOrderCommand = &cli.Command{
+	Name:      "modifyorder",
+	Usage:     "modify price and/or amount of a previously submitted order",
+	ArgsUsage: "<exchange> <asset> <pair> <order_id>",
```

Comment:
missing price and amount args

Reply:
Yes, because these two are conditional and are expected to be passed as flags, not as positional arguments.